schema.sql

```diff
@@ -12,14 +12,14 @@ ADD species TEXT;
 --owner TABLE
 
 CREATE TABLE owner  (
-  id BIGSERIAL PRIMARY KEY NOT NULL,
+  id BIGSERIAL PRIMARY KEY NOT NULL FOREIGN KEY REFERENCES animal(owner_id),
   full_name TEXT NOT NULL,
   age INT NOT NULL
 );
 
 -- species TABLE
 
 CREATE TABLE species  (
-  id BIGSERIAL PRIMARY KEY NOT NULL,
+  id BIGSERIAL PRIMARY KEY NOT NULL FOREIGN KEY REFERENCES animal(species_id),
   name_ TEXT NOT NULL
 );
```

Comment:
- [ ] As suggested by the previous reviewer you need to follow to instructions in the project requirement as it is. That is you should write queries to modify the `animals` table:
* Remove column `species`
* Add column `species_id` which is a foreign key referencing `species` table
* Add column `owner_id` which is a foreign key referencing the `owners` table

Note: Add foregin keys in `animal` table and not `species` and `owner `table